tracer/src/Datadog.Trace.Manual/SpanExtensions.ExtensionMethods.cs

```diff
@@ -0,0 +1,26 @@
+﻿// <copyright file="SpanExtensions.ExtensionMethods.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+// ReSharper disable once CheckNamespace
+
+using Datadog.Trace.SourceGenerators;
+
+namespace Datadog.Trace.ExtensionMethods;
+
+/// <summary>
+/// Extension methods for the <see cref="ISpan"/> class.
+/// </summary>
+public static class SpanExtensions
```

Comment:
What about merging this into the other `SpanExtensions`?

Reply:
Yeah, this is annoying tbh. My reason for not doing that was:
- It matches better with our Datadog.Trace APIs. But we'll likely refactor those later, so no reason not to move, except
- This extension is in a different namespace, which could be an annoying breaking change for customers to deal with, when it doesn't _really_ matter 🤷‍♂️